src/flamenco/genesis/fd_genesis_cluster.h

```diff
@@ -0,0 +1,24 @@
+#ifndef HEADER_fd_src_flamenco_genesis_fd_genesis_cluster_h
+#define HEADER_fd_src_flamenco_genesis_fd_genesis_cluster_h
+
+#include "../types/fd_types.h"
+
+/* These CLUSTER_* values must be ordered from least important to most
+   important network.  Eg, it's important that if a config has the
+   MAINNET_BETA genesis hash, but has a bunch of entrypoints that we
+   recognize as TESTNET, we classify it as MAINNET_BETA so we can be
+   maximally restrictive.  This is done by a high-to-low comparison. */
+#define FD_CLUSTER_UNKNOWN      (0UL)
+#define FD_CLUSTER_PYTHTEST     (1UL)
+#define FD_CLUSTER_TESTNET      (2UL)
+#define FD_CLUSTER_DEVNET       (3UL)
+#define FD_CLUSTER_PYTHNET      (4UL)
+#define FD_CLUSTER_MAINNET_BETA (5UL)
+
+FD_FN_PURE ulong
+fd_genesis_to_cluster( char * expected_genesis_hash );
```

Comment:
```suggestion
fd_genesis_to_cluster( char const * genesis_hash );
```

(const and it's not `expected`)